src/js/stores/AppsStore.js

```diff
@@ -128,11 +128,12 @@ function processApp(app) {
 
   app = calculateTotalResources(app);
 
-  app.status = AppStatus.RUNNING;
   if (app.deployments.length > 0) {
     app.status = AppStatus.DEPLOYING;
   } else if (app.instances === 0 && app.tasksRunning === 0) {
     app.status = AppStatus.SUSPENDED;
+  } else if (app.tasksRunning > 0) {
+    app.status = AppStatus.RUNNING;
```

Comment:
After releasing 1.0.1 we need to port this fix for this on master in https://github.com/mesosphere/marathon-ui/blob/master/src/js/stores/AppsStore.js#L131